azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunnerManager.java

```diff
@@ -147,7 +147,7 @@ public FlowRunnerManager(Props props, ExecutorLoader executorLoader,
     azkabanProps = props;
 
     // JobWrappingFactory.init(props, getClass().getClassLoader());
-    executionDirRetention = props.getLong("execution.dir.retention", executionDirRetention);
+    executionDirRetention = props.getLong("execution.dir.retention.second", executionDirRetention);
```

Comment:
This is a backward incompatible change since the property name is part of the contract.

I understand the unit makes it less convenient to config this value. Is it a big problem for you?

You are welcome to improve the code by switching to use Duration class for the internal implementation.